### PR TITLE
Changing perms to /sites/default before deleting site on drush fed.

### PR DESCRIPTION
**Pull request by jhoscar1**

### CODE REVIEW

File: fetcher.drush.inc [hunk 1]
```diff
@@ -449,6 +449,8 @@ function drush_fetcher_delete($site_name) {
   }
   try {
     $site = fetcher_get_site($site_name);
+    $default_dir = $site['site.webroot'] . '/sites/default';
+    chmod($default_dir, 0777);
```
Comment (tizzo) [inline fetcher.drush.inc]: In theory, fetcher is designed to be system agnostic. I think this should be performed via a call to (what would probably be a new method) in the [system class](https://github.com/tizzo/fetcher/blob/7.x-1.x/lib/Fetcher/System/Posix.php).

Want to take a stab at doing that? Then you could do `$site['system']->setPermission($site['site.directory'], 0777);` instead. Note also that fetcher can work with sites other than default, so you should be looking that up as done in the above.

**Comment by jhoscar1** [comment]
Responding to PR feedback

**Comment by lliss** [comment]
@jhoscar1 Can you give a little background on what this is trying to solve. I'll test it out.

**Comment by tizzo** [comment]
I think drush sets the perm of the sites dir to not be writeable on install.

**Comment by jhoscar1** [comment]
We were having an issue where, when running drush fed, fetcher would be unable to successfully delete all of the site's directories because of the permissions on the sites/default directory.

**Comment by tizzo** [comment]
Right. I think my explanation above is the why.

**Pull request merged**